integration/openshift/e2e/test_cases.txt

```diff
@@ -0,0 +1,2367 @@
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Auto-generated by gen_test_cases.sh (2020-09-18)
```

Comment:
Why is this (relatively huge) config file being stored if it can be generated by a script included on this PR?

Reply:
Good question, I did this way but I confess I'm not sure about it.

In my opinion not passing a list of tests explicitly is error prone. For this case of the OpenShift tests, there isn't any guarantee that the list will be the same between two executions because the `openshift-tests` binary might change (by mistake) so the tests list. Therefore, having a fixed list of tests ensure we are comparing the same set of tests between executions (at least nominally, because the test implementation might change after all).

The drawbacks are that it should to be re-generated every time we bump the version (see #github.com/kata-containers/runtime/pull/2969) of the test binary or when a new filter is needed; and also the file size.

About the file size, currently it simply includes all the tests from OpenShift e2e but I would like to work on narrowing the list down to just the ones which makes sense for exercising Kata Containers.

What's your opinion @jodh-intel ?